File: Plugins/Aspose_Words_Java_for_PHP/src/quickstart/helloworld/php/HelloWorld.php
<?php
/*
 * Copyright 2001-2015 Aspose Pty Ltd. All Rights Reserved.
 *
 * This file is part of Aspose.Words. The source code in this file
 * is only intended as a supplement to the documentation, and is provided
 * "as is", without warranty of any kind, either expressed or implied.
 */

class HelloWorld {

    public static function printHelloWorld() {


        // The path to the documents directory.
        $dataDir = "/usr/local/apache-tomcat-8.0.22/webapps/JavaBridge/Aspose_Words_Java_For_PHP/src/quickstart/helloworld/data/";

        // Create a blank document.
        $documentObject = new Java("com.aspose.words.Document");

        // DocumentBuilder provides members to easily add content to a document.
        $builderBoject = new Java("com.aspose.words.DocumentBuilder",$documentObject);
        // Write a new paragraph in the document with the text "Hello World!"


        $builderBoject->writeln("Hello World!");
        // Save the document in DOCX format. The format to save as is inferred from the extension of the file name.
        // Aspose.Words supports saving any document in many more formats.
        $documentObject->save($dataDir . "HelloWorld.docx");

    }

}

File: Plugins/Aspose_Words_Java_for_Ruby/lib/asposewordsjavaforruby/helloworld.rb
module Asposewordsjavaforruby
  module HelloWorld
    def initialize()
        # Create document.
        print_hello_world()
    end
        
    def print_hello_world()
        data_dir = File.dirname(File.dirname(File.dirname(__FILE__))) + '/data/quickstart/'
        
        #Create a blank document.
        document = Rjb::import('com.aspose.words.Document').new()
        
        #DocumentBuilder provides members to easily add content to a document.
        builder = Rjb::import('com.aspose.words.DocumentBuilder').new(document)
        
        #Write a new paragraph in the document with the text "Hello World!"
        builder.writeln("Hello World!")
        
        # Save the document in DOCX format. The format to save as is inferred from the extension of the file name.
        # Aspose.Words supports saving any document in many more formats.
        document.save(data_dir + "HelloWorld.docx") 
    end

  end
end
